src/pages/components/menu-buttons/accessibility.mdx

```diff
@@ -65,22 +131,81 @@ with focus (which causes focus to go somewhere else and the menu to close).
 ![illustration showing an open menu with the focus on the first item, and the arrow and Esc keys called out](images/overflow-menu-accessibility-2.png)
 
 <Caption>
-  When opened, the first item in the menu takes focus. Arrow keys move focus,
+  When opened, the first item in the menu takes focus. Arrow keys move focus and
   Esc closes the menu.
 </Caption>
 
 </Column>
 </Row>
 
+## Design recommendations
+
+Design annotations are needed for the following instances.
+
+### Labeling
+
+#### Combo button
+
+Combo button has a prop that can be used to change the tooltip label and is only
+intended to be changed to translate _'Additional actions'_ into other languages.
```

Comment:
I would double check what the content style of _'Additional actions'_ should be here according to the [IBM Style guidance](https://ibmdocs-test.dcs.ibm.com/docs/en/ibm-style?topic=punctuation-quotation-marks).